docs/src/architecture/notes.md

```diff
@@ -2,35 +2,13 @@
 A note is a way of transferring assets between accounts. A note consists of a vault and a script as shown in the diagram below.
 
 <p align="center">
-    <img src="../diagrams/protocol/note/Note.png">
+    <img src="../diagrams/architecture/note/Note.png">
 </p>
 
-A note's vault is basically the same as an account's vault. However, unlike an account, a note has a single executable script. This script is also a root of a Miden program MAST. A script is always executed in the context of a single account, and thus, may invoke account's functions. For example, a simple script could look like this:
+A note consists of:
+* a set of assets stored in a vault.
+* a script which must be executed in a context of some account to claim the assets.
+* a set of inputs which are placed onto the stack before a note's script is executed.
+* a serial number which can be used to break linkability between note hash and note nullifier.
```

Comment:
nit: maybe remove "hash" here:

* a serial number which can be used to break linkability between a note and it's nullifier.

I don't think we have introduced nullifiers in the text yet but we make reference to them here.  Maybe we should add a small note about what they are.